app/src/main/java/com/plan2meet/dto/CalendarDTO.kt

```diff
@@ -4,13 +4,13 @@ import java.time.Month
 import java.time.Year
 
 class CalendarDTO (
-    var year        : Year,
-    var month       : Month,
-    var dates       : Array<DateDTO>,
+    private var year        : Year,
```

Comment:
Updated variable type to private per the suggestion here:https://kotlinlang.org/docs/properties.html#backing-fields

Reply:
This may not be needed in a data class.  Private makes these inaccessible to other classes... and the job of a data class is to be a conduit for information between classes.  Two suggestions:
- If changing the visibility modifiers for two parameters in the constructor, be consistent and change them for all.
- Big suggestion: make this a data class.  Add they keyword "data" before "class" on line 6.